clients/spark/build.sbt

```diff
@@ -50,7 +55,10 @@ def generateCoreProject(buildType: BuildType) =
         "com.google.guava" % "guava" % "16.0.1",
         "com.google.guava" % "failureaccess" % "1.0.1",
         "org.rogach" %% "scallop" % "4.0.3",
-        "software.amazon.awssdk" % "s3" % "2.15.15",
+        // hadoop-aws provides AWS SDK at version >= 1.7.4.  So declare this
```

Comment:
nit;
```suggestion
        // hadoop-aws provides AWS SDK at version >= 1.7.4. So declare this
```

Reply:
🤷🏽 What can I say?  When They taught me to type, They said to put two spaces. The [APA agrees](https://gocolumbia.libguides.com/c.php?g=338877&p=2282177) (right now).  It is useful because it helps find the end of a sentence in typed text -- there's a single space in "A. Shaqed", so `/[.?!](  )|$/` with _two_ spaces won't find it but will find exactly the end of a sentence.

It's often considered an age thing, if you only have monospaced text it looks better.  But... computer programs are read with a monospaced font, so 2 spaces work here, too.